Description: Defaults acme-dir to /etc/home
 TODO: Put a short summary on the line above and replace this paragraph
 with a longer explanation of this change. Complete the meta-information
 with other relevant fields (see below for details). To make it easier, the
 information below has been extracted from the changelog. Adjust it or drop
 it.
 .
 acme (2.7.9-1~wheezy+bearstech2) wheezy-bearstech; urgency=medium
 .
   * Re-added --quiet support
Author: Vincent CARON <vcaron@bearstech.com>

---
The information above should follow the Patch Tagging Guidelines, please
checkout http://dep.debian.net/deps/dep3/ to learn about the format. Here
are templates for supplementary fields that you might want to add:

Origin: <vendor|upstream|other>, <url of original patch>
Bug: <url in upstream bugtracker>
Bug-Debian: https://bugs.debian.org/<bugnumber>
Bug-Ubuntu: https://launchpad.net/bugs/<bugnumber>
Forwarded: <no|not-needed|url proving that it has been forwarded>
Reviewed-By: <name and email of someone who approved the patch>
Last-Update: <YYYY-MM-DD>

--- a/acme.sh
+++ b/acme.sh
@@ -2215,9 +2215,13 @@
   #    fi
   #  fi
 
-  if [ -z "$LE_WORKING_DIR" ]; then
-    _debug "Using default home:$DEFAULT_INSTALL_HOME"
-    LE_WORKING_DIR="$DEFAULT_INSTALL_HOME"
+  if [ -z "$ACME_DIR" ] ; then
+    ACME_DIR="/etc/acme"
+  fi
+
+  # [Bearstech] Use $ACME_DIR as working directory
+  if [ -z "$LE_WORKING_DIR" ] ; then
+    LE_WORKING_DIR=$ACME_DIR
   fi
   export LE_WORKING_DIR
 
